prov/util/src/util_attr.c

```diff
@@ -622,7 +622,7 @@ int ofi_check_ep_attr(const struct util_prov *util_prov, uint32_t api_version,
 		      const struct fi_info *user_info)
 {
 	const struct fi_ep_attr *prov_attr = prov_info->ep_attr;
-	const struct fi_ep_attr *user_attr = user_info->ep_attr;
+	struct fi_ep_attr *user_attr = user_info->ep_attr;
```

Comment:
This is a 'check' call, we shouldn't modify the user's attributes.  That should be done as part of an 'alter' call.

Reply:
Thanks! Done.